src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs

```diff
@@ -40,10 +40,10 @@ public interface IInteractsWithApps : IExecuteMethod
         void ResetApp();
 
         /// <summary>
-        /// Backgrounds the current app for the given number of seconds.
+        /// Backgrounds the current app for the given number of seconds or deactivates app completely if negative number or null is given. 
         /// </summary>
-        /// <param name="seconds">a string containing the number of seconds.</param>
-        void BackgroundApp(int seconds);
+        /// <param name="seconds">a integer number containing the number of seconds or null.</param>
+        void BackgroundApp(int? seconds);
```

Comment:
passing null may be confusing. I'd rather add an overridden version of method without arguments that minimises the app

Reply:
You're right. I don't know why I didn't thought about it! I'll change that :)  👍

Reply:
Done